OpenOversight/app/templates/about.html

```diff
@@ -11,47 +11,59 @@
     <div class="hero-section no-sub">
       <div class="hero">About OpenOversight</div>
     </div>
-    <div class="vertical-padding">
-      <p class="lead">
-        <small>
-          OpenOversight is a <a href="https://lucyparsonslabs.com">Lucy Parsons Labs</a> project that aims to improve law enforcement
-          visibility and transparency using public and crowdsourced data. We maintain databases, digital galleries, and profiles of individual law enforcement officers from departments
-          across the United States that consolidate information including names, birthdates, mentions in news articles, salaries, and photographs.
-        </small>
-      </p>
-      <div class="text-center pb-50">
-        <a href="{{ url_for("main.get_officer") }}"
-           class="btn btn-lg btn-info"
-           id="try-it">Try it</a>
+    <div class="vertical-padding horizontal-padding text-center">
+      <div class="col-lg-6 text-start mx-auto">
```

Comment:
Should this be `col-xs-12 col-lg-6` or does it default to 12 if you don't have anything before a certain frame size?

Reply:
`col-12` is `width: 100%` but I can add that class to avoid confusion
![2](https://github.com/lucyparsons/OpenOversight/assets/66500457/b0914e30-bbae-42ee-9e7e-b947bebd8aee)

Reply:
If it's already defaulted to that, I have no qualms with leaving it out.

Reply:
Actually, it looks like this is what we already use today so keeping as is
https://github.com/lucyparsons/OpenOversight/blob/d989873c3640b7deed1d9bd528cacc3bc807148f/OpenOversight/app/templates/about.html#L28-L49

Reply:
I can make a ticket to standardize how we use bootstrap later on. Keeping it as is feels right to me.